config/migrations/20211216130000-correction-domains-fields.sparql

```diff
@@ -0,0 +1,122 @@
+PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX beleidsdomein: <http://kanselarij.vo.data.gift/id/beleidsdomein/>
+PREFIX beleidsveld: <http://kanselarij.vo.data.gift/id/beleidsveld/>
+PREFIX kans: <http://kanselarij.vo.data.gift/core/>
+
+PREFIX public: <http://mu.semte.ch/graphs/public>
+
+# new field
+WITH public:
+INSERT DATA {
+  beleidsveld:64634310-4c8d-46a3-ac4a-e7777a434729 a kans:Beleidsveld ;
+    a skos:Concept ;
+    skos:inScheme <http://themis.vlaanderen.be/id/concept-schema/0012aad8-d6e5-49e2-af94-b1bebd484d5b> ;
+    skos:topConceptOf <http://themis.vlaanderen.be/id/concept-schema/0012aad8-d6e5-49e2-af94-b1bebd484d5b> ;
+    skos:prefLabel "Justitie en handhaving" ;
+    ext:heeftBeleidsDomein beleidsdomein:22a39165-e17c-4a52-963a-9fa3d097907c ;
+    mu:uuid "64634310-4c8d-46a3-ac4a-e7777a434729" .
+  beleidsdomein:22a39165-e17c-4a52-963a-9fa3d097907c skos:narrower beleidsveld:64634310-4c8d-46a3-ac4a-e7777a434729 .
+};
+
+# move 4 fields from domain "Internationaal Vlaanderen" to domain "DKBUZA"
```

Comment:
De comment over `DKBUZA` is verwarrend. Hij suggereert dat er een verkeerde benaming is gebruikt voor "Kanselarij, Bestuur, Buitenlandse Zaken en Justitie (KBBJ)". Verder hier beneden zie ik weldegelijk de juiste titel.